crates/stages/src/stages/bodies.rs

```diff
@@ -748,11 +733,6 @@ mod tests {
                                 tx_transition_cursor.seek_exact(tx_id).expect("to be okay").expect("to be present").1, current_transition_id
                             );
                             current_transition_id += 1;
```

Comment:
We have "+ 1" very often in the phases, and it is not always easy to understand why.
I'd like to a note for every `+1`

perhaps this is not applicable here. not sure what the `current_transition_id` is used for, but for example in headers stage we have this a lot and it's not well documented.

Reply:
renamed and left some comments. should be more clear now